blokdata/sheet_data.py

```diff
@@ -56,5 +56,7 @@ def create_point(row):
             )),
             "extra": extra,
             "type": location_type,
+            "wheelchair": wheelchair,
+            "wifi": wifi,
```

Comment:
Gelieve daar een boolean van te maken, zodat de API taal-neutraal blijft en dit makkelijk is om te gebruiken